### PR TITLE
Ecci 518 mwt regressions

### PR DESCRIPTION
**Pull request by AnthonySGeorge**
## Call out any relevant implementation decisions
- Are there any links to back up your chosen methodology?
- Why have you taken the approach?
- Any particular problem areas?
## This PR has been tested in the following browsers
- [ ] Arc
- [ ] Edge
- [X] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] IE 11 (Windows)
- [ ] iOS Chrome
- [ ] iOS Safari
- [ ] Android Chrome
- [ ] Android Firefox
- [ ] Android default

### CODE REVIEW

**Comment by MatthewWilkes** [comment]
@AnthonySGeorge Can you please set the title of this merge request and fill in the questions in the template?

File: themes/custom/essex/css/news.css [hunk 1]
```diff
@@ -174,10 +174,13 @@ div:has(.featured-teaser),
...
 
-.featured-news__card,
+/* .featured-news__card,
 .news-card>.field__item {
     margin-top: 0;
-}
+    display: grid;
+    grid-template-columns: subgrid;
+    grid-template-rows: subgrid;
+} */
```
Comment (siliconmeadow) [inline themes/custom/essex/css/news.css:183]: Is this meant to be commented out?

**Review by siliconmeadow — CHANGES_REQUESTED** [review]

File: themes/custom/essex/css/news.css (hunk 1 above)
Comment (AnthonySGeorge) [reply themes/custom/essex/css/news.css]: Removed

**Review by siliconmeadow — APPROVED** [review]
Gorgeous!

**Pull request merged**